lib/rubocop/cop/mixin/factory_bot_mixin.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    # Helpers for detecting FactoryBot usage.
+    module FactoryBotMixin
+      extend NodePattern::Macros
+
+      def_node_matcher :factory_bot, <<~PATTERN
```

Comment:
what do you think of calling this eg `factory_bot_instantiation` or `factory_bot_build_send` or `factory_bot_send` to provide more precision? `factory_bot` has a pretty big api surface area so as named this could be read to mean "matches `FactoryBot.define`" or similar.

the existing `rails_association_hash_args` is fairly precise, as a reference point